appveyor.yml

```diff
@@ -40,6 +40,9 @@ platform:
     - x64
 
 install:
+    # Skip if we are not in a PR.
+    - cmd: if not defined APPVEYOR_PULL_REQUEST_NUMBER exit 0
```

Comment:
Will try removing all recipes. That should keep it fast without giving us an ugly exit status.